hi-ml/src/health_ml/utils/reports.py

```diff
@@ -0,0 +1,165 @@
+
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+import jinja2
+import matplotlib.pyplot as plt
+import pandas as pd
+
+IMAGE_KEY_HTML = "IMAGEPATHSHTML"
+TABLE_KEY_HTML = "TABLEKEYHTML"
+
+
+class HTMLReport:
+    """
+    Create an HTML report which Azure ML is able to render. Note that the method convert_to_pdf will not
+    apply bootstrap formatting. Very limited CSS properties are applied to the PDF
+    (see [here](https://xhtml2pdf.readthedocs.io/en/latest/reference.html#supported-css-properties))
+    """
+    def __init__(self, title: str = "Report", output_folder: str = "outputs"):
```

Comment:
can we have a constant for "outputs"? It's one of those magic folder names that we need to get righ.